pinot-common/src/main/java/com/linkedin/pinot/common/utils/CommonConstants.java

```diff
@@ -42,12 +42,14 @@
       public static class SegmentOnlineOfflineStateModel {
```

Comment:
These constants should really be pulled from the state model defintion, but we can leave that for another PR